checkov/common/bridgecrew/code_categories.py

```diff
@@ -40,6 +40,7 @@ class CodeCategoryType:
     CheckType.SERVERLESS: CodeCategoryType.IAC,
     CheckType.TERRAFORM: CodeCategoryType.IAC,
     CheckType.TERRAFORM_PLAN: CodeCategoryType.IAC,
+    CheckType.POLICY_3D: CodeCategoryType.IMAGES  # actually a combination of all code categories but currently image scannning is common to all of the 3d policies
```

Comment:
why not create another code category?

Reply:
@tsmithv11 thoughts about the code category?

Reply:
We spoke about it. I guess we can't lead with "3" so it makes sense to go with POLICY_3D

Reply:
I think we should have a new code category. Some of the policies to come won't have images.

Reply:
added